### PR TITLE
Stat tracker

### PR DESCRIPTION
**Pull request by ttakahashi1591**
## Description
This PR will add the `Stat_Tracker` class as well and the RSpec testing for it. 

## Type of change

- [ ] fix
- [x] feat
- [x] test
- [ ] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] thorough testing has been implemented if adding feature
- [x] all tests pass

### Thanks!

### CODE REVIEW

**Review by aevans27 — APPROVED** [review]
Looks good as a start for the class

File: spec/stat_tracker_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require './lib/game'
+require './lib/stat_tracker'
+# require './lib/team'
+# require './lib/game_results'
+
+RSpec.describe StatTracker do
+  before(:all) do
+    # game_path = './data/games.csv'
```
Comment (TLamb32) [inline spec/stat_tracker_spec.rb:8]: Will these paths allow us to access all data sets from within stat tracker?

**Review by lanebret85 — APPROVED** [review]
Looks great! StatTracker class and test looks well organized to move forward with. Thanks for downloading simplecov too!

**Review by TLamb32 — APPROVED** [review]
Good work. Excited to start populating this class.

**Pull request merged**